modular_skyrat/modules/more_briefcases/code/briefcases.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/storage/briefcase
+	icon = 'modular_skyrat/modules/more_briefcases/icons/briefcases.dmi'
+	inhand_icon_state = "briefcase"
+
+/obj/item/storage/briefcase/lawyer
+	icon_state = "briefcase_black"
+	inhand_icon_state = "sec-case"
+
+/obj/item/storage/briefcase/central_command
+	name = "nanotrasen briefcase"
+	icon_state = "briefcase_cc"
+	inhand_icon_state = "sec-case"
+
+/obj/item/storage/briefcase/medical
+	name = "medical briefcase"
+	icon_state = "briefcase_med"
+	inhand_icon_state = "lockbox"
+
+/obj/item/storage/briefcase/virology
+	name = "virology briefcase"
+	icon_state = "briefcase_vir"
+	inhand_icon_state = "lockbox"
+
+/obj/item/storage/briefcase/engineering
+	name = "engineering briefcase"
+	icon_state = "briefcase_eng"
```

Comment:
Are any of these actually used anywhere though?

Reply:
Not yet, but they will be eventually.